### PR TITLE
refactor: make single doctype for hackathon details

### PR DESCRIPTION
**Pull request by sumaiya2908**

### CODE REVIEW

File: mumbaihackathon_in/mumbai_hackathon/doctype/hackathon_setting/hackathon_setting.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
```
Comment (netchampfaris) [inline mumbaihackathon_in/mumbai_hackathon/doctype/hackathon_setting/hackathon_setting.json]: This should be called Hackathon Settings.

File: mumbaihackathon_in/mumbai_hackathon/doctype/hackathon_setting/hackathon_setting.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2022, Neil Lasrado and contributors
```
Comment (netchampfaris) [inline mumbaihackathon_in/mumbai_hackathon/doctype/hackathon_setting/hackathon_setting.py]: Replace Neil Lasrado with Frappe Technologies everywhere

File: mumbaihackathon_in/www/index.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {% extends "templates/hackathon.html" %}
 
-{% block title %}Mumbai Hackathon 2020 - An Open-source Hackathon{% endblock %}
+{% block title %}Mumbai Hackathon {{year}} - An Open-source Hackathon{% endblock %}
```
Comment (netchampfaris) [inline mumbaihackathon_in/www/index.html]: In registration, the year is computed from the current year, but this comes from settings. Both should be consistent.

File: mumbaihackathon_in/www/index.html [hunk 4]
```diff
@@ -9,18 +9,18 @@
 {% endblock %}
 
 {% block body %}
-    <!-- <div class="text-xl text-grey mt-8 text-center md:text-left">
-        Hack with us at the <a class="text-white hover:text-yellow no-underline" href="https://goo.gl/maps/WmifWU2Wckx">Don Bosco Institute of Technology</a> on 14th and 15th of March, and win ₹75,000 cash prize!
-    </div> -->
-    <div class="text-xl text-grey mt-8 text-center">
-        <span class="text-yellow">We're cancelling the upcoming Mumbai Hackathon in the wake of new Corona virus (COVID-19) cases in Mumbai.</span>
+    <div class="text-xl text-grey mt-8 text-center md:text-left">
+        Hack with us at the <a class="text-white hover:text-yellow no-underline" href="https://goo.gl/maps/WmifWU2Wckx">Don Bosco Institute of Technology</a> on {{details.date}}, and win ₹75,000 cash prize!
```
Comment (netchampfaris) [inline mumbaihackathon_in/www/index.html]: Make cash prize also a setting?

File: mumbaihackathon_in/mumbai_hackathon/doctype/hackathon_setting/hackathon_setting.json [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+ ],
+ "fields": [
+  {
+   "fieldname": "year",
+   "fieldtype": "Data",
+   "label": "Year"
+  },
+  {
+   "default": "0",
+   "fieldname": "disable_registration",
```
Comment (netchampfaris) [inline mumbaihackathon_in/mumbai_hackathon/doctype/hackathon_setting/hackathon_setting.json]: Does this work?

**Review by netchampfaris — CHANGES_REQUESTED** [review]

File: mumbaihackathon_in/mumbai_hackathon/doctype/hackathon_setting/hackathon_setting.json (hunk 5 above)
Comment (sumaiya2908) [reply mumbaihackathon_in/mumbai_hackathon/doctype/hackathon_setting/hackathon_setting.json]: Yes, it works.

**Pull request merged**